src/index.html

```diff
@@ -1,13 +1,54 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__photo">
+        <img
+          class="product-card__img"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+      </div>
+
+      <article class="product-card__title" data-qa="hover">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </article>
+
+      <div class="product-card__code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="product-card__rating">
```

Comment:
why it is rating block?
also
<img width="816" alt="Screenshot 2023-08-14 at 09 57 25" src="https://github.com/mate-academy/layout_product-cards/assets/53228259/eb0256ca-7240-47ce-a7d3-4a9d49862b53">